Jenkinsfile.integration

```diff
@@ -3,7 +3,10 @@
 
 def compute_node='storage-rookcheck'
 // the prefix we want to use for the rookcheck environment
-def cluster_prefix="${currentBuild.fullProjectName.replace('/', '_')}-${currentBuild.number}_"
+// def cluster_prefix="${currentBuild.fullProjectName.replace('/', '_')}-${currentBuild.number}_"
+def branch = params.BRANCH_NAME.toLowerCase()
+def cluster_prefix = 'rkck-' + \
+    branch.startsWith('pr-') ? branch.replace('-','') + params.BUILD_NUMBER + '-' : 'jenkins-'
```

Comment:
If I'm reading this right, won't pr-123 become "pr1234" where 4 is the build number. This could be confusing when we get to the actual pr-1234.

Maybe we should go for "rkck-123-4-" (drop the "pr")?

Reply:
Yeah... good catch, it is suppose to be `rkck-pr123-4-`, I would prefer to have 'pr' keyword to distinguish from the build number, though it is quite unrealistic to have over a 100 rebuilds per PR.

Reply:
Fair enough. I think that'll still be less than the current prefix workaround, so I'm happy with that.

Reply:
I have split it off to the separate PR #164